tests/upgrades/test_errata.py

```diff
@@ -215,32 +198,17 @@ def test_post_scenario_errata_count_installation(self, pre_upgrade_data):
             1. errata count, erratum list should same after satellite upgrade
             2. Installation of errata should be pass successfully
         """
-        client = pre_upgrade_data.get('rhel_client')
-        client_container_id = list(client.values())[0]
+        client_hostname = pre_upgrade_data.get('rhel_client')
         custom_repo_id = pre_upgrade_data.get('custom_repo_id')
-        product_id = pre_upgrade_data.get('product_id')
-        conten_view_id = pre_upgrade_data.get('conten_view_id')
-        product = entities.Product(id=product_id).read()
-        content_view = entities.ContentView(id=conten_view_id).read()
-        custom_yum_repo = entities.Repository(id=custom_repo_id).read()
         activation_key = pre_upgrade_data.get('activation_key')
-        host = entities.Host().search(query={'search': f'activation_key={activation_key}'})[0]
 
-        installable_errata_count = host.content_facet_attributes['errata_counts']['total']
-        tools_repo, rhel_repo = self._create_custom_rhel_tools_repos(product)
-        call_entity_method_with_timeout(product.sync, timeout=1400)
-        for repo in (tools_repo, rhel_repo):
-            content_view.repository.append(repo)
-        content_view = content_view.update(['repository'])
-        content_view.publish()
-        install_or_update_package(
-            client_hostname=client_container_id, update=True, package="katello-agent"
-        )
+        rhel_client = Broker().from_inventory(filter=f'hostname={client_hostname}')[0]
+        custom_yum_repo = target_sat.api.Repository(id=custom_repo_id).read()
+        host = target_sat.api.Host().search(query={'search': f'activation_key={activation_key}'})[0]
 
```

Comment:
I do not see sense in re-creating, re-syncing client and RHEL repos on Satellite if we did it already in the pre-test.